deploy/kubernetes/rbac.yaml

```diff
@@ -28,6 +28,9 @@ rules:
   - apiGroups: [""]
     resources: ["nodes"]
     verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
```

Comment:
Can we make these optional with a comment?  Not all drivers are going to need it and we want to reduce rbac permissions as much as possible by default.

Reply:
Ok, sounds good. Since there's no secret permission by default, should we also add a note at  https://github.com/kubernetes-csi/docs/blob/master/book/src/secrets-and-credentials.md#secrets-and-credentials to remind people to enable here?

Reply:
Yes, that's a great idea!